extension/src/main/webapp/WEB-INF/conf/gamification/storage/rule/rules-storage-configuration.xml

```diff
@@ -299,7 +299,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <external-component-plugins>
         <target-component>org.exoplatform.addons.gamification.service.setting.rule.RuleRegistry</target-component>
         <component-plugin>
-            <name>rule.AddCommenOnSpaceStream</name>
+            <name>rule.receiveCommentOnSpaceStream</name>
```

Comment:
Why do we need 2 differents rules for receive comment ? Should we have only one ?

Reply:
one for space stream and one for the network stream

Reply:
Is it possible to have one rule for both or not ? I have the feeling that we duplicate something

Reply:
We need two to have two different messages the keys are created dynamically with by concatenating rule title and the event name and the plugin name is for more consistency and readability :
Activity Stream (space) : Receive comment
Activity Stream (network) : Receive comment

all the plugins are created with this way